locales/es.yaml

```diff
@@ -290,6 +290,8 @@ wizard_community_enter_solvers_channel: Para finalizar indícame el id o nombre
 wizard_community_success: ¡Felicidades has creado tu comunidad!
 wizard_community_wrong_permissions: No tienes permisos de administrador en este grupo o canal
 wizard_exit: Has salido del modo wizard, ahora puedes escribir comandos.
+wizard_add_fiat_buy: recibir
+wizard_add_fiat_sell: pagar
```

Comment:
*enviar

Reply:
I find "pagar" a better choice of wording.

Reply:
I am native spanish speaker, let's use enviar